### PR TITLE
Fix pickle jar crafting

### LINKED ISSUE #72568: Crafting pickles is broken
Reporting client version: 514.1589

## Reproduction:
Get 10 cucumbers, 10u water, 10u salt, 1 large beaker
Craft pickle jar
Instead get jar full of 10 cucumbers and 1 beaker (none of which can fit back in the jar)

image

### PR DESCRIPTION
**Pull request by ItsTisme**
## About The Pull Request

Fixes #72568 but  not the underlying bug here. also pulls pickles made in line with the cost
afbeelding

## Why It's Good For The Game

The large beaker is used in the process it should not be added to the pickle jar, the reason for the post-qdel instead of fixing the underlying root is because this seems for some part intended, in the extend that the parts get added to the finished craft. if that isn't supposed to be the case I could rework it a bit so that reagent containers dont get added but this will very likely cause bugs in other crafting recipes

## Changelog
:cl:
fix: Pickle crafting no longer gives an empty beaker
fix: Pickle crafting costs/gives 10 cucumbers/pickles
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/fancy.dm [hunk 1]
```diff
@@ -501,7 +501,7 @@
 	name = "pickles"
 	desc = "A jar for containing pickles."
 	spawn_type = /obj/item/food/pickle
-	spawn_count = 5
+	spawn_count = 10
```
Comment (ZephyrTFA) [inline code/game/objects/items/storage/fancy.dm:504]: This isn't in the changelog or the body of your PR at all.

Comment (Autisem) [reply code/game/objects/items/storage/fancy.dm:504]: the recipie requires 10, so figured you should get 10 pickles

**Comment by Thunder12345** [comment]
But I like seasoning my pickles with smashed glass

**Comment by san7890** [comment]
Tweak isn't a valid changelog line, you should probably remove that from that portion of your PR's body.

**Review by ZephyrTFA — APPROVED** [review]

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 2]
```diff
@@ -303,6 +303,12 @@
 	result = /obj/item/storage/fancy/pickles_jar
 	category = CAT_MISCFOOD
 
+/datum/crafting_recipe/food/pickles_jar/on_craft_completion(mob/user, atom/result)
+	. = ..()
+	var/obj/item/storage/fancy/pickles_jar/jar = result
+	var/obj/item/reagent_containers/cup/beaker/large/B = locate() in jar.contents
+	qdel(B)
```
Comment (dragomagol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:310]:
```suggestion
	var/obj/item/reagent_containers/cup/beaker/large/beaker = locate() in jar.contents
	qdel(beaker)
```

Comment (ZephyrTFA) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:310]:
```suggestion
	qdel(locate(/obj/item/reagent_containers/cup/beaker/large) in jar.contents)
```
🤡

**Pull request merged**